content/tickets.rst

```diff
@@ -136,15 +138,42 @@ Below is a list of used tags:
   - These are tasks that don’t require knowledge of the product or operating system 'know-how' in advance.
 
 
+Ticket components
```

Comment:
I think that this should be done in a separate ticket... as for now we don't have explanations... and maybe we should review the componets as the current ones were created without much reasoning